src/js/steps/letters/generateOverlay.js

```diff
@@ -168,16 +168,16 @@ module.exports = function (options, $events) {
        */
       function revertCharsToOriginal(callback) {
         data.combinedLetters
-          .map((letter, index) => ({ ...letter, index }))
+          .map((letter, index) => $.merge(letter, { index }))
```

Comment:
`$.merge` is the equivalent of `_.assign` or `Object.assign` (actually, I could have just used `Object.assign`.

The code above is pretty much what you're describing, except it didn't make a copy. Does it have to be a copy? It's easy to change if it does.

```js
$.merge(letter, { index });

// ->

$.merge(letter, { index: index });

// ->

letter.index = index; return letter
```